internal/sink/kafka_connect.go

```diff
@@ -0,0 +1,57 @@
+package sink
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type KafkaConnect struct {
+	config *KafkaConnectConfig
+}
+
+type KafkaConnectConfig struct {
+	Address string
```

Comment:
If this is a URL, maybe parse it as a URL in the original config which in turn can serve as a validation: if it can't parsed as a URL, it's invalid.

Reply:
How do I parse it as a URL?